### PR TITLE
debug in VSCode

### PR DESCRIPTION
**Pull request by hsiaosiyuan0**
Hi,

Thank you for your nice work, it helps me a lot to debug v8 source code in VSCode. It's benefit from the VSCode feature [Developing inside a Container](https://code.visualstudio.com/docs/remote/containers) I just collect some the config and guides to help people to faster their debugs in VSCode

### CODE REVIEW

**Comment by bnoordhuis** [comment]
Thanks for the PR. I'm a little divided on whether to merge this or not. I appreciate that you took the time to put it together but it also feels like feature creep.

I'll leave it open for a while to see if there's demand for it. Cheers!

**Comment by gengjiawen** [comment]
Pretty helpful for me. Thanks. Din't know vscode could reach this level.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		"eamodio.gitlens",
+		"v8-torque.vscode-torque",
+		"visualstudioexptteam.vscodeintellicode"
+	],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Uncomment the next line to run commands after the container is created - for example installing git.
+	"postCreateCommand": "apt-get update && apt-get install -y git",
```
Comment (gengjiawen) [inline .devcontainer/devcontainer.json]: This should be unnecessary since git already installed in Docker image.

Comment (hsiaosiyuan0) [reply .devcontainer/devcontainer.json]: Yes, I'm sure this is a copy&paste bug 😸however it's minor and will not break the entire process

**Comment by hsiaosiyuan0** [comment]
> Thanks for the PR. I'm a little divided on whether to merge this or not. I appreciate that you took the time to put it together but it also feels like feature creep.
> 
> I'll leave it open for a while to see if there's demand for it. Cheers!

Thank you, maybe it's good enough to just leave it open 😉

**Pull request closed without merging**